nonmember_subscript_operator/main.tex

```diff
@@ -98,6 +98,33 @@ \subsection{Subscripting with scoped enumerations}
 auto value = ColorValues[Color::Red]; // now it works
 \end{lstlisting}
 
+\subsection{Compatiblity layer}
+Once P2128 is adopted, providing multi\hyp dimensional subscript operators for linear algebra (linalg) types (e.g. matrices) will be possible.
```

Comment:
Maybe we should add a "References" section and put a link to P2128 there. Or make that P2128 clickable with a link.

Reply:
Good idea - personally I'd prefer the clickable link to a static reference section. Realistically, both options exceed my TeX skills.
Looking through other papers by @mattkretz I guess, bibliography is the way to go. If he likes the example, I'l look into that...